docs/docs/adrs/adr-017-allowing-inactive-validators.md

```diff
@@ -0,0 +1,111 @@
+* Consumer chains can have a much larger validator set than the provider chain if they prefer this e.g. for decentralization reasons
+
+### Negative
+
+Allowing validators from the inactive set brings with it some additional risks.
+In general, consumer chains will now face some of the problems also faced by standalone chains. It’s reasonable to assume that the validator set on the hub has a minimal amount of operational quality due to being battle tested and decentralized, and consumer chains with validators from outside the hub active set cannot rely on this as much anymore.
+
+
+#### Sybil attacks
+
+With the restricted size of the active set today, it’s clear that the set is at least minimally competitive and it is not trivial to spin up multiple nodes as a validator.
+
+When we make the “potential validator set” much larger, we should assume that it becomes much less competitive to be part of that set, and thus trivial for single entities to control many of those validators.
+
+#### Reputational damage is not a deterrent
+
+For validators in the active set, we typically assume that if they would misbehave, they pay a large reputational cost. This represents delegators deciding to switch validators (potentially even on chains other than the one the misbehaviour happened on), and loss of credibility in the ecosystem. With the much larger active set, it seems prudent to assume that reputational damage is not a deterrent for many validators. They might only have minimal amounts of delegated stake and control most of it themselves, so they might not be deterred from performing actions that would usually bring reputational damage.
+
+#### Additional negative consequences
+* Validators outside of the active set become bonded, even if they are not validating on any consumer chains
+* The provider keeper will need to implement the staking keeper interface, and modules need to be wired up to either the staking or provider keeper, depending on whether they need the consensus or staking validator set
+* This will impact how future modules are integrated, since we will need to consider whether those modules should consider the consensus validators or the bonded validators (which other modules might assume to be the same)
+
+### Neutral
+
+## Alternative considerations
+
+### Modifying the staking module
+
+We could instead adapt the *staking module* with a similar change.
+This might be better if it turns out that the staking module active set is used in many other places.
+
+### Allowing unbonding validators to validate
+
+Instead of increasing the active set size, we could allow validators that are unbonded (but still exist on the provider) to validate consumer chains.
+For this, we would need to:
+* Modify the VSC updates to consider the set of all validators, even unbonded ones, instead of just active ones
+* Adjust our downtime jailing/equivocation slashing logic to work correctly with unbonded validators
```

Comment:
@mpoke pointed out that no matter what we do we won't be able to slash redelegations for unbonded validators, unless we keep track of redelegations on the provider. Naturally, such a solution would be substantially more complex than the one you propose. It might make sense to add a sentence here on this and on why we did not follow this alternative approach.